### PR TITLE
message reject & requeue functionality

### PR DESCRIPTION
**Pull request by cflynn07**

### CODE REVIEW

**Comment by tjmehta** [comment]
Demo is nice, but test would be better.

Can you add a test for this?

File: README.md [hunk 1]
```diff
@@ -36,7 +36,12 @@ var hermes = require('runnable-hermes').hermesSingletonFactory({
 
 var jobCallback = function (data, done) {
   // perform operation w/ data
-  done(); // remove job from RabbitMQ queue
+  done(); // remove job from RabbitMQ queue (ack)
+
+  // optionally
+    // reject = true - negative acknowledge job completion
+    // requeue = true - requeue job and resubmit to worker
+  done(/* reject, requeue */);
```
Comment (tjmehta) [inline README.md]: Positional arguments can get annoying overtime. 
They always require comments and looking at docs over-and-over.
Consider making this an options object or string (better for future support too like for nack if we ever use it).

Comment (tjmehta) [inline README.md]: Another option would be consider accepting an error to done - and just following what you think would be the most used behavior (reject and requeue upon receiving an error?)

Comment (cflynn07) [inline README.md]: agree w/ positional arguments getting annoying. Will switch to using an options object.

**Comment by anandkumarpatel** [comment]
closing. not longer needed

**Pull request closed without merging**